### PR TITLE
Refactor `BaseConstant.from_python`.

### PR DESCRIPTION
**Pull request by vpetrovykh**
Move the `from_python` dispatch into a utils function. Replace the usage
of the dispatch with calling `from_python` on specific classes where
possible.

Fixes: #1145.

### CODE REVIEW

File: edb/edgeql/utils.py [hunk 1]
```diff
@@ -119,3 +120,27 @@ def inline_anchors(
...
+MIN_INT64 = -2 ** 63
+
+
+def const_ast_from_python(val: Any) -> qlast.BaseConstant:
+    if isinstance(val, str):
+        return qlast.StringConstant.from_python(val)
+    elif isinstance(val, bool):
+        return qlast.BooleanConstant(value='true' if val else 'false')
+    elif isinstance(val, int):
+        if MIN_INT64 <= val <= MAX_INT64:
```
Comment (elprans) [inline edb/edgeql/utils.py]: I thought we decided against this, no?

Comment (vpetrovykh) [reply edb/edgeql/utils.py]: It's literally illegal then to create that `IntegerConstant` for cases outside the range. Why is it the only one that we're OK with messing up? The place where the utility function is used is generic, so technically there could be a large integer there.

Comment (vpetrovykh) [reply edb/edgeql/utils.py]: I moved it to a different utils - in `schema`, closer to `delta.py`

File: edb/schema/utils.py [hunk 2]
```diff
@@ -658,3 +659,27 @@ def _intersection_error(schema: s_schema.Schema,
...
+def const_ast_from_python(val: Any) -> qlast.BaseConstant:
+    if isinstance(val, str):
+        return qlast.StringConstant.from_python(val)
+    elif isinstance(val, bool):
+        return qlast.BooleanConstant(value='true' if val else 'false')
+    elif isinstance(val, int):
+        if MIN_INT64 <= val <= MAX_INT64:
+            return qlast.IntegerConstant(value=str(val))
+        else:
+            return qlast.BigintConstant(value=f'{val}n')
```
Comment (1st1) [inline edb/schema/utils.py]: I think we should raise an error in this case.

Comment (vpetrovykh) [reply edb/schema/utils.py]: OK, I've put a `ValueError` on that branch.

The GraphQL translator has its own ifs covering the int64/bigint case.

**Pull request merged**